charts/argo-cd/Chart.yaml

```
@@ -3,7 +3,7 @@ appVersion: v2.7.7
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.39.0
+version: 5.39.1
```

Please bump the minor version, as this is not a real bugfix
```suggestion
version: 5.40.0
```